### PR TITLE
traceid in 128bits support

### PR DESCRIPTION
**Pull request by tonyredondo**
This PR enables support for traces in 128bits

### CODE REVIEW

File: tracer/util.go [hunk 1]
```diff
@@ -75,3 +85,21 @@ func (rs *safeSource) Seed(seed int64) {
...
 	rs.Unlock()
 }
+
+func UUIDToString(uuid uuid.UUID) string {
+	if val, err := uuid.MarshalBinary(); err != nil {
+		panic(err)
+	} else {
+		return hex.EncodeToString(val)
+	}
+}
```
Comment (drodriguezhdez) [inline tracer/util.go:95]: `uuid.MarshalBinary()` method never returns an error: https://github.com/google/uuid/blob/master/marshal.go

```suggestion
func UUIDToString(uuid uuid.UUID) string {
	bytes, _ := uuid.MarshalBinary()
	return hex.EncodeToString(bytes)
}
```

or even

```suggestion
func UUIDToString(uuid uuid.UUID) string {
	return hex.EncodeToString(uuid[:])
}
```

Comment (drodriguezhdez) [reply tracer/util.go:95]: I'd add a test for this function.

File: tracer/util.go [hunk 2]
```diff
@@ -75,3 +85,21 @@ func (rs *safeSource) Seed(seed int64) {
...
+
+func StringToUUID(val string) (uuid.UUID, error) {
+	if data, err := hex.DecodeString(val); err != nil {
+		return uuid.UUID{}, err
+	} else {
+		res := uuid.UUID{}
+		err := res.UnmarshalBinary(data)
+		return res, err
+	}
+}
```
Comment (drodriguezhdez) [inline tracer/util.go:105]:
```suggestion
func StringToUUID(val string) (uuid.UUID, error) {
	if data, err := hex.DecodeString(val); err != nil {
		return uuid.UUID{}, err
	} 

	res, err := uuid.FromBytes(data)
	return res, err
}
```

Comment (drodriguezhdez) [reply tracer/util.go:105]: I'd add a test for this function, also.

File: tracer/util.go (hunk 1 above)
Comment (tonyredondo) [reply tracer/util.go]: done

File: tracer/util.go (hunk 2 above)
Comment (tonyredondo) [reply tracer/util.go]: done

**Pull request merged**